src/main/java/Duke.java

```diff
@@ -1,10 +1,201 @@
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
```

Comment:
I think the import statements can be grouped based on their package and add one blank line between the groups to enhance readability?